index.html

```diff
@@ -16,10 +16,10 @@
   </head>
   <body>
     <div id="app">
-      <app></app>
+      <tile></tile>
```

Comment:
I think you flipped the naming, `tiles` should be the container and `tile` should be the item

Reply:
👍